src/components/implementation/no_interface/vkernel/vk_types.h

```diff
@@ -11,14 +14,14 @@
 #define VM_SHM_SZ       (1<<20)	        /* Shared memory mapping for each vm = 4MB */
 #define VM_SHM_ALL_SZ   ((VM_COUNT>0)?(VM_COUNT*VM_SHM_SZ):VM_SHM_SZ)
 
-#define VM_BUDGET_FIXED 400000
-#define VM_PRIO_FIXED   TCAP_PRIO_MAX
+#define VM_FIXED_PERIOD_MS 10
+#define VM_FIXED_BUDGET_MS 5
```

Comment:
I didn't focus here at all on making parts of vkernel a generic library. Maybe, I'll add that as an issue to move such parts especially "forking" as library.